rootfs/etc/direnv/codefresh

```diff
@@ -0,0 +1,16 @@
+# This is a `BASH_ENV` script to load `direnv` configuration inside of codefresh pipelines
+# https://www.gnu.org/software/bash/manual/html_node/Bash-Startup-Files.html
+
+set -e
+
+# Prevent recursion
+unset BASH_ENV
+
+# Disable `direnv` output for scripts
+export DIRENV_LOG_FORMAT=
+
+# Allow current working directory
+direnv allow .
+
+# Process the `.envrc` and map it to `cf_export` commands
+direnv export json  | jq -r '. | to_entries[] | select (.key|test("DIRENV")|not) | "cf_export " + .key + "=" + (.value|tojson)' | bash -e
```

Comment:
I thought the general pattern in `direnv` helpers was closer to:

`source <$( ... )`

Reply:
@tamsky correct! however, in codefresh, the way to "source" is by calling `cf_export` on k=v pairs. So we're basically adapting `direnv` to also work seamlessly in codefresh.

Reply:
Thanks, that makes sense.

Any interest in formatting/wrapping lines and using `@sh` to properly escape the `.value` side?

(untested:)
```
direnv export json | 
    jq -r '. |
           to_entries[] | 
           select (.key|test("DIRENV")|not) |
           @sh "cf_export " + .key + "=" + \(.value|tojson)' | 
    bash -e
```